backend/src/main/resources/application.yml

```diff
@@ -9,3 +11,7 @@ security:
     secret : ThisIsSecretForJWTHS512SignatureAlgorithmThatMUSTHave64ByteLength
     expiration : 28800
 
+server:
+  error:
+    include-message: always
```

Comment:
What is this used for ? Is it needed ?

Reply:
to show custom error message to guide us in the front-end